cmd/main/main.go

```diff
@@ -13,14 +13,24 @@ import (
 	"github.com/t-lunch/t-lunch-backend/internal/service"
 	"github.com/t-lunch/t-lunch-backend/internal/transport"
 	"github.com/t-lunch/t-lunch-backend/pkg/gorm"
+	"github.com/t-lunch/t-lunch-backend/pkg/logger"
+	"go.uber.org/zap"
 )
 
 var cfgName string = "lunch"
 
 func main() {
+	zapLogger, err := logger.NewZapLogger()
+	if err != nil {
+		fmt.Printf("error NewZapLogger: %v\n", err)
```

Comment:
главное правило написания логов - пиши так, чтобы потом в консоли читалось как сказка
```suggestion
		fmt.Printf("failed to create new logger: %v\n", err)
```